dts/bindings/clock/st,stm32n6-cpu-clock-mux.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2022, Linaro ltd
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  STM32N6 CPU Clock
+  Describes the STM32N6 CPU clock multiplexer.
```

Comment:
Let's specifies this is the clock feeding Systick.

Reply:
Done